### DISCUSSION TITLE
Show EDAM ontology entries not on top of the page

### DISCUSSION THREAD

**Discussion by haesleinhuepf** [chanzuckerberg/napari-hub#374] [Ideas]
Hi all, 

just a suggestion about where the EDAM ontology entries are shown. At the moment, the user has to scroll down quite a bit until the actual documentation / readme of a plugin appears. I suggest to display all the useful metadata of plugins in other places (left/right), and not on top of the page. It's not urgent to work on this btw.

Happy Holidays!
Robert

![image](https://user-images.githubusercontent.com/12660498/147361155-8319ad13-349b-4d2b-94c9-8bebfff5b678.png)

**Comment by neuromusic** [comment]
Thanks @haesleinhuepf!!

Indeed, this is not ideal for a plugin with lots of labels. I'll defer to @liaprins-czi for ideas on design improvements here.

**Comment by neuromusic** [comment]
ooof, I just realized that this was bad enough for you that you pulled the categories @haesleinhuepf https://github.com/clEsperanto/napari_pyclesperanto_assistant/commit/4848e8ed14d8b86e8f4b545eb1502ca507057ec2

we'll try to improve this soon so you can take advantage of categories for pyclesperanto

**Comment by neuromusic** [comment]
just wanted to let you know @haesleinhuepf that we've redesigned the pills to better handle large numbers of categories. check it out: https://www.napari-hub.org/plugins/napari-pyclesperanto-assistant

![image](https://user-images.githubusercontent.com/1245615/163608967-eb471fe6-f14e-4566-bb17-9f5a0d84969f.png)

**Reply by haesleinhuepf** [reply]
Yes, I noticed already @neuromusic ! 🎉
It looks really cool. Fantastic work! Thanks a lot! 🙌
